spark-board-ui/src/sidebar.jsx

```diff
@@ -66,28 +66,28 @@ function TransformationDataSection({ node }) {
         case "Group":
             return <>
                 <Section title="Aggregate Expressions">
-                    <ul>{
+                    {
                         metadata.aggregate_expressions.map((exp) => (
-                            <li><Mono>{ exp }</Mono></li>
+                            <p><Mono>{ exp }</Mono></p>
```

Comment:
I don't think the `p`s are necessary if we are using `Mono`

Reply:
You're right, removed